zerver/tests/test_subs.py

```diff
@@ -719,6 +810,53 @@ def test_bad_delete_parameters(self):
         )
         self.assert_json_error(result, "delete[0] is not a string")
 
+    def test_add_or_delete_not_specified(self):
+        # type: () -> None
+        email = 'hamlet@zulip.com'
+        self.login(email)
+
+        result = self.client_patch(
+            "/api/v1/users/me/subscriptions",
+            {},
+            **self.api_auth(email)
+        )
+        self.assert_json_error(result,
+                               'Nothing to do. Specify at least one of "add" or "delete".')
+
+    def test_invalid_stream_name(self):
+        # type: () -> None
+        email = 'hamlet@zulip.com'
+        self.login(email)
+
+        invalid_stream_name = ""
```

Comment:
I'd consider using something other than the empty string here, unless you're specifically trying to test for the empty string, in which case you might want to rename the test to something like `test_empty_stream_name`.